### PR TITLE
Update "Revved up by Develocity" badge

### PR DESCRIPTION
**Pull request by runningcode**
Update "Revved up by Develocity" badge

---

Please make sure the below checklist is followed for Pull Requests.

- [ ] [All continuous integration tests](https://github.com/jhipster/generator-jhipster/actions) are green
- [ ] Tests are added where necessary
- [ ] The JDL part is updated if necessary
- [ ] [jhipster-online](https://github.com/jhipster/jhipster-online) is updated if necessary
- [ ] Documentation is added/updated where necessary
- [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/generator-jhipster/blob/main/CONTRIBUTING.md) are followed

When you are still working on the PR, consider converting it to Draft (below reviewers) and adding `skip-ci` label, you can still see CI build result at your branch.

### CODE REVIEW

**Review by DanielFran — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![NPM version][npm-image]][npm-url] [![Downloads][npmcharts-image]][npmcharts-url] [![Gitter][gitter-badge-image]][gitter-badge-url] [![Known Vulnerabilities][snyk-image]][snyk-url] [![Package Health][health-image]][health-url]
 
-[![Generator Build Status][github-actions-generator-image]][github-actions-url] [![Angular Build Status][github-actions-angular-image]][github-actions-url] [![React Build Status][github-actions-react-image]][github-actions-url] [![Vue Build Status][github-actions-vue-image]][github-actions-url] [![Revved up by Gradle Enterprise](https://img.shields.io/badge/Revved%20up%20by-Gradle%20Enterprise-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.jhipster.tech/scans)
+[![Generator Build Status][github-actions-generator-image]][github-actions-url] [![Angular Build Status][github-actions-angular-image]][github-actions-url] [![React Build Status][github-actions-react-image]][github-actions-url] [![Vue Build Status][github-actions-vue-image]][github-actions-url] [![Revved up by Develocity](https://img.shields.io/badge/Revved%20up%20by-Develocity-06A0CE?logo=Gradle&labelColor=02303A)](https://ge.jhipster.tech/scans)
```
Comment (mraible) [inline README.md:5]: The website at https://ge.jhipster.tech/scans still says Gradle Enterprise.

Comment (runningcode) [reply README.md:5]: Thanks for mentioning. That will be updated in the coming months. Product renames are a lot more difficult than badge renames :)

Comment (mraible) [reply README.md:5]: Do you have a link to the product rename announcement?

Comment (mraible) [reply README.md:5]: @bdemers Can you confirm this is legit?

Comment (bdemers) [reply README.md:5]: Confirmed!

TL;DR, it was hard to explain to folks that Gradle Enterprise was not an enterprise distribution of Gradle Build Tool 😆

**Review by bdemers — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (bdemers) [reply README.md:5]: > Do you have a link to the product rename announcement?

https://gradle.com/press-media/gradle-enterprise-is-now-develocity/

**Comment by DanielFran** [comment]
Thanks @runningcode  for the contribution and @bdemers for the confirmation

**Pull request merged**